ring-core/test/ring/middleware/session/test/cookie.clj

```diff
@@ -56,3 +56,16 @@
   (is (thrown-with-msg?
        AssertionError #"the secret key must be exactly 16 bytes"
        (cookie-store {:key (.getBytes "012345678901234567890")}))))
+
+; setup for serializing/deserializing Instant.
+(defmethod print-method java.time.Instant
+  [dt out]
+  (.write out (str "#foo/instant \"" (.toString dt) "\"")))
+(defn parseInstant [x] (java.time.Instant/parse x))
+(deftest cookie-session-custom-type
+  (let [store    (cookie-store {:readers {'foo/instant #'ring.middleware.session.test.cookie/parseInstant}})
```

Comment:
`#'ring.middleware.session.test.cookie/parseInstant` can be written more concisely as `#'parseInstant`